source/agora/node/Ledger.d

```diff
@@ -120,8 +127,8 @@ public class Ledger
         // we are only interested in the last 1008 blocks,
```

Comment:
```suggestion
        // we are only interested in the last `ValidatorCycle` blocks,
```

Reply:
Actually there are many places in the documentation which still reference `1008` as a number. We might want to fix this in another PR though, don't worry.

Reply:
I'll make a PR for it later.